src/components/carousel/carousel.js

```diff
@@ -17,7 +17,8 @@ function totalWidth() {
   return carouselList.scrollWidth;
 }
 function numItemsVisible() {
-  return Math.floor(carouselWidth() / itemWidth());
+  // Mobile portrait mode: carouselWidth() < itemWidth() so returning 1 since there is only 1 item in view
+  return Math.floor(carouselWidth() / itemWidth()) === 0 ? 1 : Math.floor(carouselWidth() / itemWidth());
```

Comment:
In mobile, the result of this is 0 which caused some of the other math to fail, i.e., `pageCount()` was dividing by 0.